source/localizable/tutorial/acceptance-test.md

```diff
@@ -0,0 +1,140 @@
+Let's think through what we want to do on the home page of our Super Rentals application.
+
+We want our application to:
+
+* List available rentals.
+* Link to information about the company.
+* Link to contact information.
+* Filter the list of rentals by city.
+
+We can represent these goals as [ember acceptance tests](../../testing/acceptance/).
+Acceptance tests interact with our app like an actual person would, but can be automated, ensuring that our app doesn't break in the future.
+
+We'll start by using Ember CLI to generate a new acceptance test:
+
+```shell
+ember g acceptance-test list-rentals
+```
+
+The command will generate the following output, showing that it created a single file called `list-rentals-test`.
+
+```shell
+installing acceptance-test
+  create tests/acceptance/list-rentals-test.js
+```
+
+Opening the new test file will reveal some boilerplate code that will try to go to the `list-rentals` route and verify that the route is loaded.
+This boiler plat code is there to guide you into your first working acceptance test.
+Since we are testing our index route, which is `/`, we'll replace occurrences of `/list-rentals` with `/`.
+
+```javascript{-6,+7,-8,+9,-12,+13}
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | list-rentals');
+
+test('visiting /list-rentals', function(assert) {
+test('visiting /', function(assert) {
+  visit('/list-rentals');
+  visit('/');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/list-rentals');
+    assert.equal(currentURL(), '/');
+  });
+});
+```
+
+Now run your test suite with `ember t -s` from the command line and you'll see one successful test!
+
+As mentioned before, this test boilerplate is just for checking the environment, so now let's replace this test with our list of goals.
+
+```/tests/acceptance/list-rentals-test.js
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | homepage');
+
+test('should list available rentals.', function (assert) {
+});
+
+test('should link to information about the company.', function (assert) {
+});
+
+test('should link to contact information.', function (assert) {
+});
+
+test('should filter the list of rentals by city.', function (assert) {
+});
+```
+
+These tests will fail, since Ember tests will always fail without assertions.
```

Comment:
And what is an assertion, pray tell? ;-)

*These tests will fail, since Ember tests will fail if we don't test for anything (known as an `assertion`)*